tutorials/install-distro.md

```diff
@@ -0,0 +1,72 @@
+# Distribution Specific Installation Instructions
+
+## Supported Versions
+Below is a compatiblity matrix between versions of CRI-O (y-axis) and distributions (x-axis)
+
+|      | Fedora 31+ | openSUSE | CentOS_8 | CentOS_8_Stream | CentOS_7 | Debian_Unstable | Debian_Testing | Debian 10 | Rasbian_10 | xUbuntu_20.04 | xUbuntu_19.10 | xUbuntu_19.04 | xUbuntu_18.04 |
+| ---- | ---------- | -------- | -------- | --------------- | -------- | --------------- | -------------- | --------- | ---------- | ------------- | ------------- | ------------- | ------------- |
+| 1.18 | x          | x        | x        | x               |          | x               | x              |           |            | x             |               |               |               |
+| 1.17 | x          | x        | x        | x               | x        | x               | x              | x         | x          | x             | x             | x             | x             |
+| 1.16 | x          | x        | x        | x               | x        | x               | x              | x         | x          | x             | x             | x             | x             |
+
+To install, choose a supported version for your operating system, and export it as a variable, like so:
+`export VERSION=1.18`
+
+
+## Installation Instructions
+
+### Fedora 31 or later
+```shell
+sudo dnf module enable cri-o:$VERSION
+sudo dnf install cri-o
+```
```

Comment:
any problem with runc/crun in f31 vs f32?

Reply:
kubernetes doesn't really yet support cgroupv2, so users will hit bigger issues